content/work/quarqnet.rst

```diff
@@ -48,3 +101,298 @@ summary: |
 
 Case Study: QuarqNet
 ====================
+
+.. callmacro:: content.macros.j2#get_quotes
+  :slug: 'work/quarqnet'
+
+.. callmacro:: content.macros.j2#rst
+  :tag: 'start'
+
+What They Needed
+----------------
+
+`Quarq`_ partnered with OddBird to
+to redesign and refactor QuarqNet,
+their existing activity tracking web application
+for professional athletes –
+making it responsive and user-friendly.
+
+.. _Quarq: https://www.quarq.com
+
+QuarqNet needed to integrate
+with Quarq's existing brand,
+both the e-commerce site
+and the packaging and printed collateral
+for the Qollector,
+Quarq's wearable tracking hardware.
+QuarqNet also matches `Quarq Race Intelligence`_,
+another Quarq web application
+OddBird had the privilege to develop.
+
+.. _Quarq Race Intelligence: https://www.quarqrace.com/
+
+.. callmacro:: content.macros.j2#rst
+  :tag: 'end'
+
+
+.. callmacro:: content.macros.j2#icon_block
+  :title: 'Planning & Discovery Phase'
+  :slug: 'work/quarqnet'
+  :data: 'planning'
+
+
+.. callmacro:: content.macros.j2#rst
+  :tag: 'start'
+
+
+UX Design & Development Process
+-------------------------------
+
+User Story
+~~~~~~~~~~
+
+OddBird's user experience
+design and development process
+always begins with a user story,
+written from the perspective of a single user.
+One of the very first user stories
+on the list for QuarqNet was:
+
+*As an athlete, I can view my activity (map & data).*
+
+Communication
+~~~~~~~~~~~~~
+
+Before diving into process details,
+a word about communication.
+At the beginning of each project,
+we establish a communication system
+for designers, developers, and clients –
+a shared Slack channel
+for daily communication and
+a schedule for weekly video meetings.
+User stories are prioritized in Pivotal Tracker
+and contain scope goals and a task list.
+To establish color patterns for QuarqNet,
+OddBird designers started with
+the existing brand colors
+and selected related user interface colors.
+These colors were communicated via Pivotal Tracker
+and coded as Sass Maps
+to establish front-end development patterns
+as well as an automated "living" `style guide`_.
+
+.. _style guide: http://quarqnet.com/styleguide/config-colors.html
+
+.. figure:: /static/images/work/quarqnet/communication.jpg
+   :class: extend-large
+   :alt: user story in Pivotal Tracker and Sass color maps
+
+Initial Sketches
+~~~~~~~~~~~~~~~~
+
+.. image:: /static/images/work/quarqnet/activity.jpg
+   :class: extend-left img-shadow
+   :alt: initial sketch of activity map and data
```

Comment:
Looks like there isn't enough space between the bottom of the Sketches sections/images and the headings for the sections that follow. I may be able to solve this by making the images smaller?